src/rgw/rgw_rest.cc

```diff
@@ -673,7 +673,7 @@ void end_header(struct req_state* s, RGWOp* op, const char *content_type,
   /* do not send content type if content length is zero
      and the content type was not set by the user */
   if (force_content_type ||
-      (!content_type &&  s->formatter->get_len()  != 0) || s->err.is_err()){
```

Comment:
@lnsyyj the idea is that we need to explicitly set the content type when sending the response at RGWOp_User_Create::execute()